modules/database/main.tf

```diff
@@ -11,10 +11,12 @@ resource "aws_db_instance" "database" {
   instance_class         = "db.t2.micro"
   identifier             = "${var.namespace}-db-instance"
   name                   = "pets"
-  username               = "admin"
+  username               = "<some_random_username>"
```

Comment:
**Tenable.cs remediation**
 
**[MEDIUM]: Ensure master username does not use commonly predicted usernames for Amazon Relational Database Service (Amazon RDS) instances**
 
**Vulnerability**
Using usernames (example - admin) that can be easily predicted can lead to easy username spray and brute-force attack.
 
**Remediation**
In AWS Console - 
1. Sign in to the AWS Console and go to the AWS RDS Console.
2. In the RDS Dashboard, click on Databases.
3. Select Modify to modify the instance of your choice.
4. Change the username if it is something common like 'awsuser'.

In Terraform -
1. In the aws_db_instance resource, set the username field; this field is required unless creating the database as a replica or building from a snapshot.
2. Use a valid string that is not a commonly used, or system generated, name such as awsuser, admin, pgadmin, mysqladmin, or sqladmin.

References:
[https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_CreateDBInstance.html](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_CreateDBInstance.html)
[https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/db_instance#username](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/db_instance#username)

source: tenablecs-